libbeat/publisher/queue/queue.go

```diff
@@ -47,6 +47,8 @@ type Metrics struct {
 
 	//UnackedConsumedEvents is the count of events that an output consumer has read, but not yet ack'ed
 	UnackedConsumedEvents opt.Uint
+	//UnackedConsumedEvents is the count of bytes that an output consumer has read, but not yet ack'ed
+	UnackedConsumedBytes opt.Uint
 
 	//OldestActiveTimestamp is the timestamp of the oldest item in the queue.
 	OldestActiveTimestamp common.Time
```

Comment:
Can you remove `OldestActiveTimestamp` from this struct since we are not implementing it, since it requires too many changes to the internals of both queues?

Reply:
That'll require making changes in `elastic-agent-shipper`, so we may want to do that in a separate PR?

Reply:
Yes let's do a follow up PR to clean this up.